sources/auth.c

```diff
@@ -344,6 +364,10 @@ int od_auth_frontend(od_client_t *client)
 	case OD_RULE_AUTH_BLOCK:
 		od_auth_frontend_block(client);
 		return -1;
+    case OD_RULE_AUTH_PAM:
```

Comment:
This case and `OD_RULE_AUTH_PAM` should be removed as described above to avoid confusion between auth methods methods. PAM should be supported both for clear_text and md5 as well

Reply:
But the standard protocol postgreSQL only supports cleartext in PAM